packages/terra-form-input/src/Input.jsx

```diff
@@ -114,6 +120,7 @@ class Input extends React.Component {
     const formInputClassNames = cx([
       'form-input',
       { 'form-error': isInvalid },
+      { 'form-incomplete': (isIncomplete && required && !isInvalid) },
```

Comment:
Did we hear back from UX about whether `isIncomplete` and `isInvalid ` are mutually exclusive?

Reply:
Yes - needs to be mutually exclusive in preparation for future UI updates that pairs styling to be similar (error, incomplete, and others).  If there are components that "need" both, should be reviewed by UX and guided into an appropriate resolution.

Reply:
@BenBoersma -- we should however add tests for when both are applied.

Reply:
Added example and test for when both `isInvalid` and `isIncomplete` are set: https://github.com/cerner/terra-core/pull/2670/commits/f3184124d28bce916fe72b76f0f19fa0aea8c0fd